roles/openshift_examples/tasks/main.yml

```diff
@@ -36,6 +36,7 @@
   unarchive:
     src: "{{ copy_examples_mktemp.stdout }}/openshift-examples.tar"
     dest: "{{ examples_base }}/"
+    mode: 0755
```

Comment:
I noticed that the previous task was creating a directory with mode `0755`. On a subsequent you would not expect to see that directory created OR CHANGED in the results. However, the directory was actually showing changed every time. The mode starts out as `0755` from that earlier task, but then the `unarchive` task comes up. Here's where it gets interesting.

Git [does not store permissions for directories](http://stackoverflow.com/a/1071274), just executable bits for flat files. When the archive is created the root directory was created with `0765` permissions, that is the mode I see for the examples directory on my local checkout. Probably a system `umask` side-effect.

When `unarchive` ran on the remote side it will change the examples destination directory to the `0765` mode.

You see the cycle now? On the next run the *Create the remote OpenShift examples directory* task runs and changes the mode back to `0755` again. Then the unarchive happens and changes the mode back to `0765`. This update to `unarchive`, the addition of `mode` is intended to prevent the constant flippy-flappy.

Ultimately how we copy the examples over could probably be reevaluated. We stopped using the synchronize module because it's realllllly slow for large numbers of files (like our examples directories). 

In an earlier upgrade run I ran **ara** against you would see results like this:

```json
    "diff": {
        "after": {
            "mode": "0755", 
            "path": "/usr/share/openshift/examples"
        }, 
        "before": {
            "mode": "0765", 
            "path": "/usr/share/openshift/examples"
        }
    }, 
```

That task no longer shows up in the **changed** results any more with the added`mode` setting.